packages/dev/s2-docs/pages/react-aria/getting-started.mdx

```diff
@@ -0,0 +1,186 @@
+
+import {InstallCommand} from '../../src/InstallCommand';
+import {Command} from '../../src/Command';
+import {BundlerSwitcher, BundlerSwitcherItem} from '../../src/BundlerSwitcher';
+import {StepList, Step, Counter} from '../../src/Step';
+import {ShadcnCommand} from '../../src/ShadcnCommand';
+import {StarterKits} from '../../src/StarterKits';
+import docs from 'docs:react-aria-components';
+import '../../tailwind/tailwind.css';
+
+export const section = 'Getting started';
+export const tags = ['introduction', 'installation'];
+export const description = 'Install and build your first component';
+
+# Getting started
+
+<PageDescription>How to install React Aria and build your first component.</PageDescription>
+
+## Install
+
+Install React Aria with your preferred package manager.
+
+<InstallCommand pkg="react-aria-components" />
+
+## Quick start
+
+The documentation for each component includes vanilla CSS and [Tailwind](https://tailwindcss.com) examples. Copy and paste these into your project and make them your own. You can also download each example as a ZIP or open in CodeSandbox or StackBlitz.
+
+<ExampleSwitcher>
+  ```tsx render docs={docs.exports.Select} links={docs.links} props={[]} type="vanilla" files={["starters/docs/src/Select.tsx", "starters/docs/src/Select.css"]} showCoachMark
+  "use client";
+  import {Select, SelectItem} from 'vanilla-starter/Select';
+
+  <Select label="Favorite animal">
+    <SelectItem>Aardvark</SelectItem>
+    <SelectItem>Cat</SelectItem>
+    <SelectItem>Dog</SelectItem>
+    <SelectItem>Kangaroo</SelectItem>
+    <SelectItem>Panda</SelectItem>
+    <SelectItem>Snake</SelectItem>
+  </Select>
+  ```
+
+  ```tsx render docs={docs.exports.Select} links={docs.links} props={[]} type="tailwind" files={["starters/tailwind/src/Select.tsx"]} showCoachMark
+  "use client";
+  import {Select, SelectItem} from 'tailwind-starter/Select';
+
+  <Select label="Favorite animal">
+    <SelectItem>Aardvark</SelectItem>
+    <SelectItem>Cat</SelectItem>
+    <SelectItem>Dog</SelectItem>
+    <SelectItem>Kangaroo</SelectItem>
+    <SelectItem>Panda</SelectItem>
+    <SelectItem>Snake</SelectItem>
+  </Select>
+  ```
+
+</ExampleSwitcher>
+
+### shadcn CLI
+
+Each example can also be installed with the [shadcn](https://ui.shadcn.com/docs/cli) CLI. This will add the component source code, styles, and all dependencies to your project automatically.
+
+<ShadcnCommand registryUrl="vanilla/Select.json" />
+
+### Storybook starter kits
+
+If you're building a full component library, download a pre-built [Storybook](https://storybook.js.org/) starter kit. These include every component in a standalone development environment.
+
+<StarterKits />
+
+### Working with AI
+
+Use the menu at the top of each page in the docs to open or copy it into your favorite AI assistant. We also have an [MCP server](mcp.html) which can be used directly in your IDE, and [llms.txt](../llms.txt) which can help AI agents navigate the docs.
```

Comment:
pretty minor but wonder if we could/should add a coach mark to this menu button as well when the user is on this page specifically. Might be overkill tbh